packages/py-api/py_api/controllers/getting_logs_controller.py

```diff
@@ -0,0 +1,44 @@
+import pathlib
+from os.path import exists
+from traceback import format_exc
+from typing import Final
+
+from fastapi import APIRouter
+from py_api.environment import IS_OFFLINE
+from starlette.responses import FileResponse, JSONResponse
+
+router = APIRouter()
+LOGFILE_PATH: Final = f"{pathlib.Path(__file__).parent.resolve().parent}/shared/logfile.log"

```

Comment:
Let's import Path from path lib instead of having it like this

Reply:
pathlib.Path(__file__).parent.resolve().parent
                                                               ^^^^^^^
                                                         Are you sure you need this?

Reply:
Oh yes, you do. My bad. You're in a child directory.

Reply:
> Let's import Path from path lib instead of having it like this

Fixed